src/compiler/parser/index.js

```diff
@@ -538,10 +538,16 @@ function processAttrs (el) {
             name = camelize(name)
           }
           if (modifiers.sync) {
+            syncGen = genAssignmentCode(value, `$event`)
+            addHandler(
+              el,
+              `update:${kebabize(name)}`,
```

Comment:
The `hyphenate` function might be a better fit here.

Reply:
Good point! 🙂 I forgot that one existed. @shortdiv, can you update to use `hyphenate` instead?